e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js

```diff
@@ -29,7 +29,7 @@ function verifyMessageAutocomplete(index, type = 'user') {
     cy.get('#suggestionList .mentions__name').eq(index).should('be.visible').and('have.class', 'suggestion--selected').within((el) => {
         if (type === 'user') {
             cy.wrap(el).invoke('text').then((text) => {
-                const username = text.replace('- ', '').replace('@', '').replace('(', '').replace(')', '').toLowerCase();
+                const username = text.replace('- ', '').replace('@', '').replace('(you)', '').replace('(', '').replace(')', '').toLowerCase();
```

Comment:
I've got this failing tests:
![Screen Shot 2020-03-11 at 8 46 25 AM](https://user-images.githubusercontent.com/5334504/76371598-b360a400-6375-11ea-9021-15577d60f939.png)

Reply:
Odd. Tests pass for me locally. However, when I run against cypress test server, I get similar error. I just updated the files to fix errors against cypress server (can't repro locally). Please check changes again. Thanks.

Reply:
Btw, failures not related to original fix but updated nevertheless